### PR TITLE
Feature/dump

### PR DESCRIPTION
**Pull request by daitamae**
- Purpose: to add dump feature creating a local csv file with all the metadata stored in the device.
- Sample downloaded file:
[samples.csv](https://github.com/spirabr/PWA-App/files/10932814/download.1.1.csv)
- New button:
new_button

### CODE REVIEW

**Comment by daitamae** [comment]
Tested locally and feature seems to be working just fine, as the sample csv file shows.
I am not quite sure why build is failing in CI, could you help me with that?

File: src/views/Upload.vue [hunk 1]
```diff
@@ -68,8 +78,33 @@ export default {
...
     },
+    dumpAllSamples() {
+      const samples = this.$refs.uploadCards.map((card) => card.dumpSample());
+      
+      if(samples.size == 0){
+        return;
+      }
+
+      const fields = Object.keys(samples[0]);
+      const replacer = function(nothing, value) { return value === null ? '' : value; };
```
Comment (Franwernke) [inline src/views/Upload.vue:89]: What do you think about turning this into an arrow function?

Comment (Franwernke) [reply src/views/Upload.vue:89]: Also, better use a "dangling underscore" (_) to represent unused parameters

**Review by Franwernke — APPROVED** [review]
LGTM!

File: src/views/Upload.vue (hunk 1 above)
Comment (daitamae) [reply src/views/Upload.vue]: Done

**Comment by daitamae** [comment]
For the record, the build error happening during CI pipeline was due to the Node.js version.

**Pull request merged**